go/grpcweb/options.go

```diff
@@ -99,3 +101,9 @@ func WithWebsocketOriginFunc(websocketOriginFunc func(req *http.Request) bool) O
 		o.websocketOriginFunc = websocketOriginFunc
 	}
 }
+
+func WithAllowNonRootResource(allowNonRootResources bool) Option {
```

Comment:
Ah, something I missed before, please add a comment explaining what this is useful, ideally with an example.

Reply:
Will get on that shortly 👍

Reply:
^^